pkg/velaql/providers/query/tree.go

```diff
@@ -677,6 +678,9 @@ func iteratorChildResources(ctx context.Context, cluster string, k8sClient clien
 			clusterCTX := multicluster.ContextWithClusterName(ctx, cluster)
 			items, err := listItemByRule(clusterCTX, k8sClient, resource, *parentObject, specifiedFunc, rules.DefaultGenListOptionFunc)
 			if err != nil {
+				if meta.IsNoMatchError(err) || runtime.IsNotRegisteredError(err) {
```

Comment:
By analogy, are there any other errors that need to be ignored?

Reply:
Have considered this issue. IMO ignoring all error isn't proper.